.github/workflows/checks.yml

```
@@ -0,0 +1,13 @@
+name: checks
+run-name: Spot check for errors
+on:
+  - push
+jobs:
+  checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: curl -L https://nixos.org/nix/install | sh
```

```suggestion
      - uses: cachix/install-nix-action@v20
```

It works better than the regular installer for GH Actions.